Simplified.xcodeproj/project.pbxproj

```diff
@@ -6928,7 +6928,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = 13;
+				CURRENT_PROJECT_VERSION = 14;
```

Comment:
We already have 3.7.0(14) on Firebase, should this be 15 instead?

Reply:
you're right, I missed it somehow... fixed